charts/metallb/templates/NOTES.txt

```diff
@@ -0,0 +1,11 @@
+
+MetalLB is now running in the cluster.
+{{- if .Values.configInline }}
+LoadBalancer Services in your cluster are now available on the IPs you
+defined in MetalLB's configuration. To see IP assignments,
+try `kubectl get services`.
```

Comment:
maybe print the inline config ?

Also if someone changed the ConfigMap directly and the next helm upgrade "revert" the config, is there a way to print the old and new config ?

It's more a limitation of Helm but having to type the repo and version just to change the current config without upgrading is not user friendly at all, so not a fan of configInline when `kubectl edit` is all we need ;)

Reply:
I can display the inline config.  I don't think there's a way to get diffs between releases/upgrades.

Reply:
So [helm-diff](https://github.com/databus23/helm-diff) is a plugin that allows you to check diffs between releases/upgrades. There is talk of this plugin being merged into helm core [here](https://github.com/databus23/helm-diff/issues/211). 

However, I don't think this chart has to worry about manual changes being reverted upon a helm upgrade. I think it is well understood that manual changes won't persist if helm is part of your workflow and you have not updated helm upgrade params.

Reply:
> I think it is well understood that manual changes won't persist if helm is part of your workflow and you have not updated helm upgrade params.

you assume people know how things are working and are not following blindly procedures and first google reponse :P

I agree this is not the job of the chart, but it would have been nice. I'm thinking about the message displayed by apt when default config changes, do you want to keep local or maintainer version, see a diff, edit, ..., this is really a good UX IMO.

Reply:
I think helm diff is a reasonable solution, especially when combined with tools like [Praqma/helmsman](https://github.com/Praqma/helmsman).

Reply:
Updated NOTES.txt template